autoload/reply/repl/ghci.vim

```diff
@@ -24,6 +24,25 @@ function! s:repl.extract_input_from_terminal_buf(lines) abort
     return input
 endfunction
 
+function! s:repl.get_command() abort
+    return self.get_base_command() + self.get_var('command_options', [])
+endfunction
+
+functio! s:repl.get_base_command() abort
+  let result = [self.executable()]
+  if self.executable() == 'stack'
+    let result += ['exec', 'ghci']
+  endif
+  return result
+endfunctio
```

Comment:
I don't think the `if` statement is necessary. User can use `stack` for `ghci` as follows with the current master branch:

```vim
let g:reply_repl_ghci_executable = 'stack'
let g:reply_repl_ghci_command_options = ['exec', 'ghci']
```

Please try not to add special case.

I think these changes in `repl/ghci.vim` are not necessary.